schema/deploy/shipping/views.sql

```diff
@@ -3043,6 +3049,10 @@ create or replace view shipping.uw_reopening_ehs_reporting_v1 as
   , encounters.live_other_uw as lives_with_uw_students_or_employees
   , encounters.uw_apt_yesno as lives_in_uw_apartment
   , encounters.tier as study_tier
+  , encounters.uw_housing_resident
+  , encounters.on_campus_2x_week
+  , encounters.uw_housing_group
+  , encounters.alerts_off
```

Comment:
Slightly off top, will these fields get added to the EHS REDCap project?

Reply:
I thought it would be good to have these new fields available in the EH&S view in case EH&S do end up wanting to see them in their REDCap project. There's no plan yet to add them, though.

Reply:
Ah okay. `uw_housing_resident` and `on_campus_2x_week` are calculated from existing fields, so I wonder if they are redundant. Not sure if `uw_housing_group` and `alerts_off` make any sense outside of the HCT project.

Reply:
I could imagine EH&S wanting to know which day of the week a UW Housing participant gets invited for testing.
I could also see them wanting them to know if a PT doesn't get alerts from REDCap, potentially.
We don't have to put them in the transfer project. I put them in the view for us to have if we need them.